simulate/sam.go

```diff
@@ -13,9 +14,10 @@ import (
 )
 
 // IlluminaPairedSam generates a pair of sam reads randomly distributed across the input ref.
-func IlluminaPairedSam(refName string, ref []dna.Base, numPairs, readLen, avgFragmentSize int, avgFragmentStdDev float64, flatErrorRate float64, binomialAlias numbers.BinomialAlias, out *fileio.EasyWriter, bw *sam.BamWriter, bamOutput bool) {
+func IlluminaPairedSam(refName string, ref []dna.Base, numPairs, readLen, avgFragmentSize int, avgFragmentStdDev float64, flatErrorRate float64, ancientErrorRate float64, flatBinomialAlias numbers.BinomialAlias, ancientBinomialAlias numbers.BinomialAlias, geometricParam float64, out *fileio.EasyWriter, bw *sam.BamWriter, bamOutput bool, deaminationDistributionSlice []int) {
```

Comment:
Right now ancient DNA damage of a molecule is independent across the fwd and rev reads, but in reality, I think if a position is damaged, and read by both the forward and reverse reads, both should show the damage.  Is that correct?  If so, I think this would be important to capture.

Reply:
(Note that this refers to fwd and rev reads from Illumina of the same strand of DNA, not the 2 strands of 1 DNA molecule)

Reply:
Resolved in latest commit